pkg/koordlet/metricsadvisor/collectors/podresource/pod_resource_collector.go

```diff
@@ -162,7 +162,15 @@ func (p *podResourceCollector) collectPodResUsed() {
 			return
 		}
 
-		metrics = append(metrics, cpuUsageMetric, memUsageMetric)
+		memWithPageCacheUsageValue := memStat.MemWithPageCacheUsage()
```

Comment:
To not increase the agent overhead, pod metrics of usage with page cache should not be collected by default. Instead, it needs to be collected when the memory collecting policy in the NodeMetric is `usageWithPageCache`. Or we can add a new collector plugin for cache metrics collection (just like the kilded collector).